packages/ember-data/lib/system/model/states.js

```diff
@@ -417,6 +417,10 @@ createdState.uncommitted.rollback = function(record) {
   record.transitionTo('deleted.saved');
 };
 
+createdState.uncommitted.pushedData = function(record) {
+  record.transitionTo('loaded.saved');
+};
+
```

Comment:
Interesting. I'm curious why this doesn't end up putting records created with data directly into the `loaded.saved` state, since `buildRecord` should call `pushedData`, which (seems to) call this transition. I'm actually unsure when this transition get invoked at all since the only two paths I can identify are the the one above and `push` => `_load` => `setupData` => `pushedData`, which shouldn't affect the created state.

Damn the `DS.Model` state machine is difficult to reason about :stuck_out_tongue_closed_eyes: